guides/doc-Administering_Red_Hat_Satellite/topics/adding_rhel_system_roles.adoc

```diff
@@ -1,16 +1,34 @@
 [[chap-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Managing_Ansible_Roles]]
 == Managing Ansible Roles
 
-In {Project}, you can import Ansible roles and Red{nbsp}Hat Enterprise Linux system roles to help with automation of routine tasks. Ansible is enabled by default on {Project} and {SmartProxy}.
+In {Project}, you can import Ansible roles to help with automation of routine tasks. Ansible is enabled by default only on {Project}.
 
-If you use custom or third party Ansible roles, you must add the roles to the `/etc/ansible/roles` directory of the {SmartProxy} or {Project} where you want to use them.
+[[chap-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Configuring_your_Deployment_to_Run_Ansible_Roles]]
+=== Configuring your Deployment to Run Ansible Roles
 
-You must import the Ansible roles into {ProjectServer} from the `/etc/ansible/roles` directory before you can use them.
+Complete this procedure to configure your {Project} deployment to run Ansible roles.
+
+.Procedure
+
+. Add the roles to the `/etc/ansible/roles` directory on the {Project} and all {SmartProxies} from where you want to use the roles. If you want to use custom or third party Ansible roles, ensure to configure an external version control system to synchronize roles between {Project} and {SmartProxies}.
+
+. On all {SmartProxies} that you want to use to run Ansible roles on hosts, enable the Ansible plug-in:
++
+----
+# satellite-installer --scenario capsule \
+--enable-foreman-proxy-plugin-ansible true
```

Comment:
Style point: I always like to indent on multi line statements
```suggestion
  --enable-foreman-proxy-plugin-ansible true
```

However, I think it's not common in this repo.

Reply:
Yeah, we do not do that because sometimes options get very long and these two extra spaces get annoying then.